OAuth/ResourceOwner/GeniusResourceOwner.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware.Info <opensource@hardware.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HWI\Bundle\OAuthBundle\OAuth\ResourceOwner;
+
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * GeniusResourceOWner.
```

Comment:
not needed

Reply:
Added it because it was in almost every resource owner. Removed in 6dd9ad8.